model/span/span.go

```diff
@@ -56,7 +65,7 @@ type Span struct {
 	TransactionId string
 }
 
-func DecodeSpan(input interface{}, err error) (*Span, error) {
+func DecodeSpan(input interface{}, err error) (transform.Transformable, error) {
```

Comment:
You need to add setting the `Timestamp` to the `DecodeSpan` method, as for v2 the spans are not sent within `transactions` any more. 
There was an agreement that if the Timestamp is not set it should be set to the time "that the HTTP request from the agent was first received. "

Reply:
Instead of setting the request time during decoding, I'm now doing it during transformation. I added a request time to the transformation context which is used to set the time if it's absent.